account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940

```diff
@@ -0,0 +1,17 @@
+:20:6450374100
```

Comment:
In the https://github.com/OCA/bank-statement-import/blob/8.0/account_bank_statement_import/tests/test_import_file.py#L90 there is the path predefined to the folder "test_files" from the module path...